src/Components/CardContainer/Card/Card.js

```diff
@@ -11,9 +11,12 @@ export class Card extends Component {
 
   render() {
     const { poster } = this.props;
+    let style = { 
```

Comment:
I like the use of the poster here to set the individual backgrounds for each card.